mash_scripts/katello/4.10.0/katello-candlepin-4.10-el8.mash

```diff
@@ -0,0 +1,12 @@
+[katello-candlepin-4.10-el8]
+rpm_path = %(arch)s/os/Packages
+repodata_path = %(arch)s/os/
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = katello-candlepin-4.10-el8
+inherit = True
+strict_keys = True
+keys = bdcc85ae
```

Comment:
Same here

Reply:
Looks like we had the gpg from 4.9 in the config file. I just updated the mash and the config.